src/Components/Header/Header.css

```diff
@@ -111,17 +120,20 @@
   background-color: #D43C0D;
   color: #ffffff;
   border: none;
-  border-radius: 25px;
+  border-radius: 2em;
```

Comment:
I know em's are a more responsive unit, wondering why initially you used px? Is it easier just to get something on the page and easy enough to change it as it gets closer to the final product? Just picking your brain so I can be more like you lol